resources/migrations/000_migrations.yaml

```diff
@@ -3939,4 +3939,52 @@ databaseChangeLog:
                   type: text
               - column:
                   name: ended_at
-                  type: DATETIME
\ No newline at end of file
+                  type: DATETIME
+  - changeSet:
+      id: 69
+      author: senior
+      comment: 'Added 0.27.0'
+      remarks: 'Add columns to the pulse table for alerts'
+      changes:
+        - addColumn:
+            tableName: pulse
+            columns:
+              - column:
+                  name: alert_condition
+                  type: varchar(254)
+                  remarks: 'Condition (i.e. "rows" or "goal") used as a guard for alerts'
+                  constraints:
+                    nullable: true
+              - column:
+                  name: alert_description
+                  type: varchar(254)
+                  remarks: 'Text included in alert notifications'
+                  constraints:
+                    nullable: true
+              - column:
+                  name: alert_first_only
+                  type: boolean
+                  remarks: 'True if the alert should be disabled after the first notification'
+                  constraints:
+                    nullable: true
+              - column:
+                  name: alert_above_goal
+                  type: boolean
+                  remarks: 'For a goal condition, alert when above the goal'
+                  constraints:
+                    nullable: true
+  - changeSet:
+      id: 71
```

Comment:
@attekei bumped this from 70. changeset 69 above wasn't released, I could collapse this with the one above and it goes away entirely. People working on this branch thought would need to do some manual `databasechangelog` work though

Reply:
👍  on collapsing these into a single migration

Everyone who's already loaded the branch should probably be able to just do `lein run migrate force` and that should fix things automatically